composer.json

```diff
@@ -8,11 +8,18 @@
         {
             "type": "vcs",
             "url": "https://github.com/unl/phpunltemplates.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/unl/phpdwtparser.git"
         }
     ],
     "require": {
-        "pear-unl/UNL_DWT": "*",
+        "unl/php-dwt-parser": "1.0.0-rc.2",
```

Comment:
Should be okay to remove the requirement. It will be pulled from other requirements.